code/datums/traits/good.dm

```diff
@@ -169,3 +169,16 @@
 	var/obj/item/autosurgeon/gloweyes/gloweyes = new(get_turf(H))
 	H.equip_to_slot(gloweyes, SLOT_IN_BACKPACK)
 	H.regenerate_icons()
+
+/datum/quirk/bloodpressure
+	name = "Synthetic blood"
```

Comment:
Better to say something akin to Big Bones, more bone marrow rather than synthetics